general/apero_reduction_interface/simple_ari.py

```diff
@@ -1175,6 +1175,7 @@ def get_time_series_parameters(self):
             num_obs_ext = str(np.sum(obs_mask_ext))
             num_obs_tcorr = str(np.sum(obs_mask_tcorr))
             # get the seeing for this observation directory
+            # TODO: nanmean for all below?
```

Comment:
I think it would make sense to change mean to nanmean (I'll do it once I have an OK someone more familiar with ARI), unless we expect some report to show NaN on the mean when one of the values is NaN.

Reply:
Keeping `np.mean` for now.